Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV DEBIAN_FRONTEND noninteractive
 # Install packages
 RUN apt-get update -qq && apt-get install -y \
     locales \
+    zip \
```

Comment:
If you merge a master branch it should get rid of this commit as it's already there.